drivers/counter/counter_ll_stm32_rtc.c

```diff
@@ -275,10 +275,15 @@ static int rtc_stm32_init(struct device *dev)
 
 #else /* CONFIG_COUNTER_RTC_STM32_CLOCK_LSE */
 
-#ifndef(CONFIG_SOC_SERIES_STM32F4X)
+#if !defined(CONFIG_SOC_SERIES_STM32F4X) &&	\
+	!defined(CONFIG_SOC_SERIES_STM32F2X) &&	\
```

Comment:
Let's remove the F2, if F2 is not yet supported.
We'll add and issue to state driver should be ported on F2. It'll be less confusing.

Reply:
Got it

Reply:
Oh, I shall correct myself, not F2, but F1